woocommerce/woocommerce-gutenberg-products-block
# Cart i2: view switcher shared to children

Implements `__experimentalExposeToChildren` so that the view switcher is passed down to all children toolbars. When switching, the view is changed and the parent is selected to avoid the selection of a non-visible block.

Fixes #4812

NOTE: Only the top 2 levels of inner blocks will show the toolbar item, unless you first implement https://github.com/WordPress/gutenberg/pull/35078 This makes it so all levels show the toolbar item.

### How to test the changes in this Pull Request:

1. Insert Cart i2 Block
2. Toggle between empty/full cart views
3. Select inner blocks. Confirm toolbar item is visible.
4. Toggle between empty/full cart views. Parent block should be selected after a switch.
